File: FlightService/Models/Passenger.cs
namespace FlightService.Model
{
    public class Passenger
    {
        public int Id { get; set; }
        public string FirstName { get; set; } = String.Empty;
        public string LastName { get; set; } = String.Empty;
        public int Age { get; set; }
        public string Job { get; set; } = String.Empty;

        public string Email { get; set; } = String.Empty;
        public int ConfirmationNumber { get; set; }

        public ICollection<Flight>? Flights { get; set; } = new List<Flight>();

        // Could possibily add count for number of flights.

    }
}

File: FlightServiceWeb/flight-service/src/models/passenger.ts
import Flight from "./flight"

interface Passenger {
    Id: number;
    FirstName: string;
    LastName: string;
    Age: number;
    Job: string;
    Email: string;
    ConfirmationNumber: number;
    Flights: Flight[];
}
// export so other ts extensions can use.
export default Passenger;